src/scripts/underground/Underground.ts

```diff
@@ -58,6 +58,30 @@ class Underground implements Feature {
                     GameHelper.createArray(50, 250, 50), GameConstants.Currency.diamond),
                 GameHelper.createArray(0, 10, 2)
             ),
+            new UndergroundUpgrade(
+                UndergroundUpgrade.Upgrades.SurveyCharge_Cost, 'Survey Charge Cost', 5,
+                AmountFactory.createArray(
+                    GameHelper.createArray(50, 250, 50), GameConstants.Currency.diamond),
+                GameHelper.createArray(0, 5, 1)
+            ),
+            new UndergroundUpgrade(
+                UndergroundUpgrade.Upgrades.SurveyCharge_Efficiency, 'Survey Charge Efficiency', 4,
```

Comment:
I think we should leave this as just `survey` rather than `survery charge`,
As otherwise it will cause overflow on smaller screens.
```suggestion
                UndergroundUpgrade.Upgrades.Survey_Cost, 'Survey Cost', 5,
                AmountFactory.createArray(
                    GameHelper.createArray(50, 250, 50), GameConstants.Currency.diamond),
                GameHelper.createArray(0, 5, 1)
            ),
            new UndergroundUpgrade(
                UndergroundUpgrade.Upgrades.Survey_Efficiency, 'Survey Efficiency', 4,
```

Reply:
Done. Changed to Survey.